amperity_datagrid/source/bridge_snowflake.rst

```diff
@@ -0,0 +1,346 @@
+
+.. bridge-snowflake-sync-amperity-links-end
+
+
+.. _bridge-snowflake-sync-amperity-get-details:
+
+Get details
+--------------------------------------------------
+
+.. bridge-snowflake-sync-amperity-get-details-start
+
+Before you can create inbound sharing between Snowflake and Amperity you will need to collect the following information.
+
+.. list-table::
+   :widths: 10 90
+   :header-rows: 0
+
+   * - .. image:: ../../images/steps-arrow-off-black.png
+          :width: 60 px
+          :alt: Requirement 1.
+          :align: left
+          :class: no-scaled-link
+     - The organization name and account name for your brand's |destination-name| account.
+
+       .. include:: ../../shared/credentials_settings.rst
+          :start-after: .. credential-snowflake-organization-start
+          :end-before: .. credential-snowflake-organization-end
+
+       .. include:: ../../shared/credentials_settings.rst
+          :start-after: .. credential-snowflake-account-name-start
+          :end-before: .. credential-snowflake-account-name-end
+
+       .. tip:: To `find your organization name and account name <https://docs.snowflake.com/en/user-guide/admin-account-identifier#finding-the-organization-and-account-name-for-an-account>`__ |ext_link| use **Snowsight**.
+
+          Open the account selector and browse to the account for which Amperity Bridge will be configured. Hover over the account name to view additional details, and then copy the account identifier.
+
+          The copied identifier contains both organization and account name in the format `organization-name.account-name`.
+
+
+   * - .. image:: ../../images/steps-arrow-off-black.png
+          :width: 60 px
+          :alt: Requirement 2.
+          :align: left
+          :class: no-scaled-link
+     - .. include:: ../../shared/credentials_settings.rst
+          :start-after: .. credential-snowflake-region-start
+          :end-before: .. credential-snowflake-region-end
+
+
+   * - .. image:: ../../images/steps-arrow-off-black.png
+          :width: 60 px
+          :alt: Requirement 3.
+          :align: left
+          :class: no-scaled-link
+     - Before you can configure Amperity Bridge for data sharing with Snowflake you must :ref:`create a share in Snowflake <bridge-snowflake-sync-amperity-configure-snowflake-create-share>`, add a secure share identifier to that share, identify the correct account locator, and then add the :ref:`Amperity account locator ID <bridge-snowflake-sync-amperity-configure-snowflake-account-locator>` to the share.
+
+       Use the `CURRENT_ACCOUNT <https://docs.snowflake.com/en/sql-reference/functions/current_account>`__ |ext_link| argument in |destination-name| to return the locator ID for your Snowflake account.
+
+       .. important:: Secure data sharing should :ref:`only share secure views <bridge-snowflake-sync-amperity-configure-snowflake-secure-views>` with Amperity.
```

Comment:
This is a bit ambigous and sounds like they should _only_ share secure views which isn't true - in reality they can share other tables and secure objects with Amperity - but we want to discourage sharing _non-secured views_ specifically.

I would say something like:

When sharing views via secure data sharing, Snowflake strongly recommends only sharing secure views. While it is possible to share data in [non-secured views](https://docs.snowflake.com/en/user-guide/data-sharing-views#label-sharing-non-secure-view-limitations), this is not a reccommended practice.